armi/reactor/tests/test_composites.py

```diff
@@ -124,6 +134,16 @@ def test_iterComponents(self):
         self.assertIn(self.thirdGen, list(self.container.iterComponents()))
 
     def test_getChildren(self):
+        """Test the get children method.
+
+        .. test:: Components are a physical part of the reactor.
```

Comment:
Same comment here about the test not actually verifying that components are a part of the reactor.

Reply:
I have a test for that part of the req in test_reactors.py:HexReactorTests:test_coreSfp()